x/evm/types/msg_create_deploy_token.go

```diff
@@ -10,13 +10,14 @@ import (
 )
 
 // NewCreateDeployTokenRequest is the constructor for CreateDeployTokenRequest
-func NewCreateDeployTokenRequest(sender sdk.AccAddress, chain string, asset Asset, tokenDetails TokenDetails, minAmount sdk.Int) *CreateDeployTokenRequest {
+func NewCreateDeployTokenRequest(sender sdk.AccAddress, chain string, asset Asset, tokenDetails TokenDetails, minAmount sdk.Int, address Address) *CreateDeployTokenRequest {
 	return &CreateDeployTokenRequest{
 		Sender:       sender,
 		Chain:        utils.NormalizeString(chain),
 		Asset:        asset,
 		TokenDetails: tokenDetails,
 		MinAmount:    minAmount,
+		Address:      address,
```

Comment:
Dont' we need to update `Validate` with a check for this new field?

Reply:
Address is of type `byte[20]` and therefore there's nothing to be validated.